config/main.py

```diff
@@ -1642,12 +1642,54 @@ def breakout(ctx, interface_name, mode, verbose, force_remove_dependencies, load
 
     """ Load config for the commands which are capable of change in config DB """
     cm = load_configMgmt(verbose)
+
+    """ Save Port OIDs Mapping Before Deleting Port"""
+    dataBase = SonicV2Connector(host="127.0.0.1")
+    if_name_map, if_oid_map = port_util.get_interface_oid_map(dataBase)
+
     """ Delete all ports if forced else print dependencies using configMgmt API """
     final_delPorts = [intf for intf in del_intf_dict.keys()]
     delete_Ports(cm, final_delPorts, force_remove_dependencies, verbose)
 
-    # TODOFIX:  Check ASIC DB whether interfaces got deleted or not
-    time.sleep(5)
+    """
+     Internal Function:  Check ASIC DB whether interfaces present or not
+     ports: List of ports
+     presence: If False, Make sure Ports are not present, return False otherwise.
+               If True, Make sure Ports are present, return False otherwise.
+    """
+    def check_Ports_AsicDb(db, ports, presence, portMap):
```

Comment:
Can we please move this function out of breakout function? we loose context while reading.

Reply:
I can move it on top of breakout function.